modules/pam_time/pam_time.c

```diff
@@ -83,7 +83,7 @@ _pam_parse (const pam_handle_t *pamh, int argc, const char **argv, const char **
     }
 
 #ifdef VENDOR_PAM_TIME_CONF
-    if (*conffile == PAM_TIME_CONF) {
+    if (strcmp(*conffile, PAM_TIME_CONF) == 0) {
```

Comment:
Another option would be to make the default value to be an static const char default_conf[] = PAM_TIME_CONF; And then you can keep using the pointer comparison.